lnd_test.go

```diff
@@ -1696,6 +1696,13 @@ func assertMinerBlockHeightDelta(t *harnessTest,
 // channel where the funding tx gets reorged out, the channel will no
 // longer be present in the node's routing table.
 func testOpenChannelAfterReorg(net *lntest.NetworkHarness, t *harnessTest) {
+	// Skip test for neutrino, as we cannot disconnect the miner at will.
```

Comment:
Can't we just restart the node and connect it to the new miner?

Reply:
Added TODO.